src/helpers/azure.ts

```diff
@@ -6,9 +6,34 @@ import { DefaultAzureCredential } from "@azure/identity";
 import { AdditionalPolicyConfig } from "@azure/core-client";
 import { debug, isDebug } from "@actions/core";
 
+import { getOptionalEnumInput } from "./input";
+
 const userAgentPrefix = "gh-azure-bicep-deploy";
 const dummySubscriptionId = "00000000-0000-0000-0000-000000000000";
 
+function getEndpoint(): string | undefined {
```

Comment:
Could we move the reading of config here? https://github.com/Azure/bicep-deploy/blob/246f83361529f7509304027ffc49c2647d6923ec/src/config.ts#L53-L59

This makes it easier to unit test.

Reply:
See https://github.com/Azure/bicep-deploy/blob/246f83361529f7509304027ffc49c2647d6923ec/test/config.test.ts#L252 and https://github.com/Azure/bicep-deploy/blob/246f83361529f7509304027ffc49c2647d6923ec/test/config.test.ts#L138-L140 for examples testing this.